.github/composite-actions/setup-new-version/action.yml

```diff
@@ -27,9 +27,16 @@ runs:
       run: cp "/usr/local/lib/libantlr4-runtime.so.4.9.3" ~/${{ inputs.pg_new_dir }}/lib/
       shell: bash
 
+    - name: Build PostGIS Extension
+      id: build-postgis-extension
+      if: always() && steps.copy-antlr.outcome == 'success'
+      uses: ./.github/composite-actions/build-postgis-extension
+      with:
+        install_dir: ${{ inputs.pg_new_dir }}
+
     - name: Build Extensions
       id: build-extensions-new
-      if: always() && steps.copy-antlr.outcome == 'success'
```

Comment:
is there risk here of breaking the entire test workflow if postgis source files are unreachable? (ie if postgis site or geos site is unreachable)

Reply:
It is a very rare case if source files are unreachable, we use it in building other extensions as well such as tds_fdw, so, there shouldn't be a bigger risk here